lib/driverProviders/driverProvider.ts

```diff
@@ -28,17 +31,28 @@ export class DriverProvider {
     return this.drivers_.slice();  // Create a shallow copy
   }
 
+  getBPUrl() {
+    return `http://localhost:${this.bpRunner.port}`;
```

Comment:
I think localhost is the right thing to do here, since BP is binding to localhost.